src/vivarium_dcpn_hypertension_mgt/components/treatment_algo.py

```diff
@@ -41,3 +44,65 @@ def get_rx_fill(self, index) -> pd.Series:
     def get_appt_followup(self, index) -> pd.Series:
         return self.rand_appt_followup.get_draw(index) < self.df_ad_data.appt_ad.loc[index]
 
+
+class MeasuredSBP:
+    @property
+    def name(self):
+        return "high_systolic_blood_pressure"
```

Comment:
hmm I know this will mess up things in the setup but let's maybe name this `high_systolic_blood_pressure_measurement`. We can have a `self.risk = 'high_systolic_blood_pressure'` for things to use in setup but I think that will make it clearer and prevent any confusion with the high sbp risk component itself